### ISSUE TITLE
Distance calculation

### ISSUE THREAD

**Issue by hesamd** [dtao/nearest-color#9]
The following code
```JavaScript
const colors = {
    'red.sd': '#f00',
    'yellow.PNG': '#ff0',
    'blue.PNG': '#00f'
};

const nearest = nearestColor.from(colors);
console.log('nearest', nearest('#ff0'));
```
Prints
```
nearest { name: 'yellow.PNG',
  value: '#ff0',
  rgb: { r: 255, g: 255, b: 0 },
  distance: 441.6729559300637 }
```
While the answer is correct, shouldn't the distance be equal to 0?

**Comment by dtao** [comment]
You are right, that definitely looks wrong!

I should fix that.

**Comment by dtao** [comment]
This is fixed in 0.4.2.
